packages/fast-components-react-msft/src/divider/examples.data.ts

```diff
@@ -0,0 +1,20 @@
+import {ICategoryItemProps} from "@microsoft/fast-development-site-react";
+import {IGenericExample} from "../examples";
+import {IManagedClasses} from "@microsoft/fast-jss-manager-react";
+import Divider from "./index";
+import {DividerRole, IDividerHandledProps} from "@microsoft/fast-components-react-base";
+
+export default {
+    name: "divider",
+    component: Divider,
+    data: [
+        {
```

Comment:
can you try using:

`role: void 0`

An empty object seems like a mistake, but I understand what you are doing